--- vice-3.7.1/src/Makefile.am.orig	2022-12-28 00:56:56.000000000 +0100
+++ vice-3.7.1/src/Makefile.am	2023-06-27 19:30:24.598179035 +0200
@@ -515,10 +515,10 @@ infocontrib.h: $(srcdir)/buildtools/geni
 
 # also update the version/date in the readme
 # FIXME: this is not the exact right place for this
-	@$(SHELL) $(srcdir)/buildtools/updatereadme.sh $(top_srcdir)/README $(top_srcdir)/configure.ac $(top_srcdir)/src/vicedate.h
+	@bash $(srcdir)/buildtools/updatereadme.sh $(top_srcdir)/README $(top_srcdir)/configure.ac $(top_srcdir)/src/vicedate.h
 # also update the version/date in the index.html
 # FIXME: this is not the exact right place for this
-	@$(SHELL) $(srcdir)/buildtools/updateindex.sh $(top_srcdir)/doc/html/index.html $(top_srcdir)/configure.ac $(top_srcdir)/src/vicedate.h
+	@bash $(srcdir)/buildtools/updateindex.sh $(top_srcdir)/doc/html/index.html $(top_srcdir)/configure.ac $(top_srcdir)/src/vicedate.h
 
 arch_gtk3_lib = $(top_builddir)/src/arch/gtk3/libarch.a
 arch_gtk3_widgets_lib = $(top_builddir)/src/arch/gtk3/widgets/libwidgets.a
