diff -Nru vice-1.13.orig/src/arch/unix/archdep.c vice-1.13/src/arch/unix/archdep.c
--- vice-1.13.orig/src/arch/unix/archdep.c	2003-04-24 21:00:14.000000000 +0200
+++ vice-1.13/src/arch/unix/archdep.c	2003-11-09 13:56:07.000000000 +0100
@@ -115,6 +115,14 @@
     return home;
 }
 
+const char *archdep_home_etc_path(void)
+{
+    char *home_etc = getenv("HOME_ETC");
+
+    if (home_etc) return home_etc;
+		return archdep_home_path;
+}
+
 char *archdep_default_sysfile_pathlist(const char *emu_id)
 {
     static char *default_path;
@@ -124,7 +132,7 @@
         const char *home_path;
 
         boot_path = archdep_boot_path();
-        home_path = archdep_home_path();
+        home_path = archdep_home_etc_path();
 
         /* First search in the `LIBDIR' then the $HOME/.vice/ dir (home_path)
            and then in the `boot_path'.  */
@@ -161,7 +169,7 @@
 {
     const char *home;
 
-    home = archdep_home_path();
+    home = archdep_home_etc_path();
     return util_concat(home, "/.vice/vicerc", NULL);
 }
 
@@ -169,7 +177,7 @@
 {
     const char *home;
 
-    home = archdep_home_path();
+    home = archdep_home_etc_path();
     return util_concat(home, "/.vice/fliplist-", machine_name, ".vfl", NULL);
 }
 
@@ -179,7 +187,7 @@
     const char *home;
     char *viceuserdir;
 
-    home = archdep_home_path();
+    home = archdep_home_etc_path();
 
     viceuserdir = util_concat(home, "/.vice", NULL);
 
diff -Nru vice-1.13.orig/src/arch/unix/archdep.h vice-1.13/src/arch/unix/archdep.h
--- vice-1.13.orig/src/arch/unix/archdep.h	2002-12-02 20:19:02.000000000 +0100
+++ vice-1.13/src/arch/unix/archdep.h	2003-11-09 13:56:41.000000000 +0100
@@ -76,6 +76,7 @@
 #define ARCHDEP_RAWDRIVE_DEFAULT "/dev/fd0"
 
 extern const char *archdep_home_path(void);
+extern const char *archdep_home_etc_path(void);
 
 #endif
 
diff -Nru vice-1.13.orig/src/arch/unix/uicommands.c vice-1.13/src/arch/unix/uicommands.c
--- vice-1.13.orig/src/arch/unix/uicommands.c	2003-09-07 19:30:12.000000000 +0200
+++ vice-1.13/src/arch/unix/uicommands.c	2003-11-09 13:57:50.000000000 +0100
@@ -478,7 +478,7 @@
 
 static UI_CALLBACK(load_quicksnap)
 {
-    char *fname = util_concat(archdep_home_path(), "/", VICEUSERDIR, "/",
+    char *fname = util_concat(archdep_home_etc_path(), "/", VICEUSERDIR, "/",
         machine_name, ".vsf", NULL);
 
     if (!ui_emulation_is_paused())
@@ -510,7 +510,7 @@
 
 static UI_CALLBACK(save_quicksnap)
 {
-    char *fname = util_concat(archdep_home_path(), "/", VICEUSERDIR, "/",
+    char *fname = util_concat(archdep_home_etc_path(), "/", VICEUSERDIR, "/",
         machine_name, ".vsf", NULL);
 
     interrupt_maincpu_trigger_trap(save_snapshot_trap, (void *)fname);
