diff -Nur vice-1.8.orig/configure.in vice-1.8/configure.in
--- vice-1.8.orig/configure.in	Sun Jan 20 20:59:10 2002
+++ vice-1.8/configure.in	Sat Mar 30 11:00:32 2002
@@ -96,6 +96,7 @@
 
   AM_CONDITIONAL(USE_MIDAS_SOUND, test x"$with_midas" = "xyes")
   AM_CONDITIONAL(HAVE_RS232, false)
+  AM_CONDITIONAL(USE_VSID, false)
 
   dnl We always have a joystick and mouse on MSDOS.
   AC_DEFINE(HAS_JOYSTICK)
@@ -112,7 +113,9 @@
 
   AC_DEFINE(HAS_JOYSTICK)
   AC_DEFINE(HAVE_MOUSE)
+  AM_CONDITIONAL(USE_MIDAS_SOUND, false)
   AM_CONDITIONAL(HAVE_RS232, false)
+  AM_CONDITIONAL(USE_VSID, false)
 
   AC_TRY_LINK([#include <windows.h>], [LARGE_INTEGER li; return (int) li.QuadPart],
   	 AC_DEFINE(HAS_LONGLONG_INTEGER),)
@@ -129,7 +132,9 @@
 
   AC_DEFINE(HAS_JOYSTICK)
   AC_DEFINE(HAVE_MOUSE)
+  AM_CONDITIONAL(USE_MIDAS_SOUND, false)
   AM_CONDITIONAL(HAVE_RS232, false)
+  AM_CONDITIONAL(USE_VSID, false)
 
 else
 
@@ -189,6 +194,7 @@
 
   dnl This is only possible on Unix (for now).
   AC_DEFINE(HAVE_RS232)
+  AM_CONDITIONAL(USE_MIDAS_SOUND, false)
   AM_CONDITIONAL(HAVE_RS232, true)
 
   AC_DEFINE(USE_VSID)
@@ -423,9 +429,9 @@
 
     dnl Test for libresid.a and resid/sid.h
     AC_CHECK_LIB(resid, resid_version_string,
-      AC_CHECK_HEADER(resid/sid.h,
+      [AC_CHECK_HEADER(resid/sid.h,
         ,
-        AC_MSG_ERROR([reSID header file not found])),
+        AC_MSG_ERROR([reSID header file not found])]),
       AC_MSG_ERROR([reSID library not found]))
 
     AC_DEFINE(HAVE_RESID)
@@ -814,12 +820,12 @@
                  [ AC_DEFINE(DWORD_IS_LONG)
                    AC_MSG_RESULT(unsigned long) ], AC_MSG_RESULT(unsigned int))
 
-  x64_LDFLAGS="-mwindows"
-  x128_LDFLAGS="-mwindows"
-  xvic_LDFLAGS="-mwindows"
-  xpet_LDFLAGS="-mwindows"
-  xplus4_LDFLAGS="-mwindows"
-  xcbm2_LDFLAGS="-mwindows"
+  x64LDFLAGS="-mwindows"
+  x128LDFLAGS="-mwindows"
+  xvicLDFLAGS="-mwindows"
+  xpetLDFLAGS="-mwindows"
+  xplus4LDFLAGS="-mwindows"
+  xcbm2LDFLAGS="-mwindows"
 
   X64_WINRES="\$(top_builddir)/src/arch/win32/resc64.o"
   X128_WINRES="\$(top_builddir)/src/arch/win32/resc128.o"
@@ -870,12 +876,12 @@
 
 fi
 
-AC_SUBST(x64_LDFLAGS)
-AC_SUBST(x128_LDFLAGS)
-AC_SUBST(xvic_LDFLAGS)
-AC_SUBST(xpet_LDFLAGS)
-AC_SUBST(xplus4_LDFLAGS)
-AC_SUBST(xcbm2_LDFLAGS)
+AC_SUBST(x64LDFLAGS)
+AC_SUBST(x128LDFLAGS)
+AC_SUBST(xvicLDFLAGS)
+AC_SUBST(xpetLDFLAGS)
+AC_SUBST(xplus4LDFLAGS)
+AC_SUBST(xcbm2LDFLAGS)
 
 AC_SUBST(X64_WINRES)
 AC_SUBST(X128_WINRES)
@@ -975,7 +981,6 @@
 AC_SUBST(ARCH_INCLUDES)
 AC_SUBST(ARCH_LIBS)
 AC_SUBST(ARCH_DIR)
-AC_SUBST(LIBOBJS)
 AC_SUBST(SOUND_USS_OBJ)
 AC_SUBST(SOUND_SGI_OBJ)
 AC_SUBST(SOUND_HPUX_OBJ)
diff -Nur vice-1.8.orig/src/Makefile.am vice-1.8/src/Makefile.am
--- vice-1.8.orig/src/Makefile.am	Wed Nov 14 16:13:30 2001
+++ vice-1.8/src/Makefile.am	Sat Mar 30 10:58:13 2002
@@ -206,7 +206,7 @@
 video_lib = $(top_builddir)/src/video/libvideo.a
 xcbm2_lib = $(top_builddir)/src/cbm2/libcbm2.a
 
-common_libs = $(rs232_ldadd) @ARCH_LIBS@ @LIBOBJS@ @READLINE@ @RESID_LIBS@ @SOUND_LIBS@
+common_libs = $(rs232_ldadd) @ARCH_LIBS@ @READLINE@ @RESID_LIBS@ @SOUND_LIBS@
 
 bin_PROGRAMS = x64 x128 xvic xpet xplus4 xcbm2 c1541 petcat
 
@@ -232,7 +232,7 @@
 x64_SOURCES = $(base_sources) $(maincpu_sources)
 x64_LDADD = $(x64_libs) @X_LIBS@
 x64_DEPENDENCIES = $(x64_libs)
-x64_LDFLAGS = @x64_LDFLAGS@
+x64_LDFLAGS = @x64LDFLAGS@
 
 .PHONY: x64-all
 x64-all: \
@@ -274,7 +274,7 @@
 x128_SOURCES = $(base_sources)
 x128_LDADD = $(x128_libs) @X_LIBS@
 x128_DEPENDENCIES = $(x128_libs)
-x128_LDFLAGS = @x128_LDFLAGS@
+x128_LDFLAGS = @x128LDFLAGS@
 
 .PHONY: x128-all
 x128-all: \
@@ -313,7 +313,7 @@
 xvic_SOURCES = $(base_sources) $(maincpu_sources)
 xvic_LDADD = $(xvic_libs) @X_LIBS@
 xvic_DEPENDENCIES = $(xvic_libs)
-xvic_LDFLAGS = @xvic_LDFLAGS@
+xvic_LDFLAGS = @xvicLDFLAGS@
 
 .PHONY: xvic-all
 xvic-all: \
@@ -347,7 +347,7 @@
 xpet_SOURCES = $(base_sources) $(maincpu_sources)
 xpet_LDADD = $(xpet_libs) @X_LIBS@
 xpet_DEPENDENCIES = $(xpet_libs) 
-xpet_LDFLAGS = @xpet_LDFLAGS@
+xpet_LDFLAGS = @xpetLDFLAGS@
 
 .PHONY: xpet-all
 xpet-all: \
@@ -381,7 +381,7 @@
 xplus4_SOURCES = $(base_sources)
 xplus4_LDADD = $(xplus4_libs) @X_LIBS@
 xplus4_DEPENDENCIES = $(xplus4_libs)
-xplus4_LDFLAGS = @xplus4_LDFLAGS@
+xplus4_LDFLAGS = @xplus4LDFLAGS@
 
 .PHONY: xplus4-all
 xplus4-all: \
@@ -417,7 +417,7 @@
 xcbm2_SOURCES = $(base_sources)
 xcbm2_LDADD = $(xcbm2_libs) @X_LIBS@
 xcbm2_DEPENDENCIES = $(xcbm2_libs) 
-xcbm2_LDFLAGS = @xcbm2_LDFLAGS@
+xcbm2_LDFLAGS = @xcbm2LDFLAGS@
 
 .PHONY: xcbm2-all
 xcbm2-all: \
diff -Nur vice-1.8.orig/src/arch/unix/readline/Makefile.am vice-1.8/src/arch/unix/readline/Makefile.am
--- vice-1.8.orig/src/arch/unix/readline/Makefile.am	Sun Sep 12 21:17:22 1999
+++ vice-1.8/src/arch/unix/readline/Makefile.am	Sat Mar 30 10:58:13 2002
@@ -10,8 +10,6 @@
 
 noinst_LIBRARIES = libreadline.a
 
-EXTRA_DIST = README
-
 libreadline_a_SOURCES = \
 	editline.c \
 	editline.h \
