--- vice-1.22/src/arch/unix/x11/gnome/x11ui.c~	2007-08-04 15:24:43.000000000 +0200
+++ vice-1.22/src/arch/unix/x11/gnome/x11ui.c	2007-08-13 11:04:42.000000000 +0200
@@ -424,9 +424,9 @@
     gtk_init(&fake_argc, &fake_args);
 
     /* set X11 fontpath */
-    if (access(PREFIX "/lib/vice/fonts/fonts.dir", R_OK) == 0)
+    if (access(PREFIX "/share/vice/fonts/fonts.dir", R_OK) == 0)
     {
-	const char *cmd = "xset fp+ " PREFIX "/lib/vice/fonts";
+	const char *cmd = "xset fp+ " PREFIX "/share/vice/fonts";
 	
 	if (system(cmd) != 0)
 	    fprintf(stderr, _("Can't add fontpath `%s'.\n"), cmd);
@@ -1404,7 +1404,7 @@
 	ui_dispatch_events();
 
 	/* remove fontpath, Don't care about result */
-	system("xset fp- " PREFIX "/lib/vice/fonts");
+	system("xset fp- " PREFIX "/share/vice/fonts");
 
         lib_free(s);	
 	exit(0);
--- vice-1.22/src/arch/unix/archdep.h~	2007-04-29 20:50:17.000000000 +0200
+++ vice-1.22/src/arch/unix/archdep.h	2007-08-13 11:06:44.000000000 +0200
@@ -106,11 +106,7 @@
 extern const char *archdep_home_path(void);
 
 /* Define the default system directory (where the ROMs are).  */
-#ifdef __NetBSD__
 #define LIBDIR          PREFIX "/share/vice"
-#else
-#define LIBDIR          PREFIX "/lib/vice"
-#endif
 
 #if defined(__FreeBSD__) || defined(__NetBSD__)
 #define DOCDIR          PREFIX "/share/doc/vice"
--- vice-1.22/configure.in~	2007-08-05 21:52:39.000000000 +0200
+++ vice-1.22/configure.in	2007-08-13 11:09:09.000000000 +0200
@@ -2049,20 +2049,20 @@
 		   [Where do we want to install the executable?])
 
 dnl Path for keymaps, ROMs etc.
-case "$host_os" in
-  netbsd*)
-    VICEDIR="$PREFIX/share/vice";
-    AM_CONDITIONAL(ALTERNATE_DOCDIR, true)
-    ;;
-  freebsd*)
-    VICEDIR="$PREFIX/lib/vice";
-    AM_CONDITIONAL(ALTERNATE_DOCDIR, true)
-    ;;
-  *)
-    VICEDIR="$PREFIX/lib/vice";
+#case "$host_os" in
+#  netbsd*)
+#    VICEDIR="$PREFIX/share/vice";
+#    AM_CONDITIONAL(ALTERNATE_DOCDIR, true)
+#    ;;
+#  freebsd*)
+#    VICEDIR="$PREFIX/lib/vice";
+#    AM_CONDITIONAL(ALTERNATE_DOCDIR, true)
+#    ;;
+#  *)
+    VICEDIR="/usr/share/vice";
     AM_CONDITIONAL(ALTERNATE_DOCDIR, false)
-    ;;  
-esac
+#    ;;  
+#esac
 AC_SUBST(VICEDIR)
 
 if false ; then
--- vice-1.22/data/fonts/Makefile.am~	2006-01-30 22:48:17.000000000 +0100
+++ vice-1.22/data/fonts/Makefile.am	2007-08-13 11:10:43.000000000 +0200
@@ -1,4 +1,4 @@
-fontsdir = $(VICEDIR)/fonts
+fontsdir = /usr/share/fonts/misc
 
 fonts_DATA = \
 	vice-cbm.bdf
@@ -7,7 +7,6 @@
 
 install: install-data-am
 	@echo "Preparing fontdir..." ; \
-	cd $(DESTDIR)$(fontsdir) ; \
+	cd $(DESTDIR)$(datadir)/fonts/misc ; \
 	bdftopcf -o vice-cbm.pcf vice-cbm.bdf ; \
-	rm vice-cbm.bdf ; \
-	mkfontdir .
+	rm vice-cbm.bdf
