--- vice-1.21/src/arch/unix/x11/gnome/x11ui.c~	2007-03-04 22:47:42.000000000 +0100
+++ vice-1.21/src/arch/unix/x11/gnome/x11ui.c	2007-07-06 19:51:34.000000000 +0200
@@ -425,9 +425,9 @@
     gtk_init(&fake_argc, &fake_args);
 
     /* set X11 fontpath */
-    if (access(PREFIX "/lib/vice/fonts/fonts.dir", R_OK) == 0)
+    if (access(PREFIX "/share/vice/fonts/fonts.dir", R_OK) == 0)
     {
-	const char *cmd = "xset fp+ " PREFIX "/lib/vice/fonts";
+	const char *cmd = "xset fp+ " PREFIX "/share/vice/fonts";
 	
 	if (system(cmd) != 0)
 	    fprintf(stderr, _("Can't add fontpath `%s'.\n"), cmd);
@@ -1435,7 +1435,7 @@
 	ui_dispatch_events();
 
 	/* remove fontpath, Don't care about result */
-	system("xset fp- " PREFIX "/lib/vice/fonts");
+	system("xset fp- " PREFIX "/share/vice/fonts");
 
         lib_free(s);	
 	exit(0);
--- vice-1.21/src/arch/unix/archdep.h~	2005-03-28 21:10:20.000000000 +0200
+++ vice-1.21/src/arch/unix/archdep.h	2007-07-06 19:53:16.000000000 +0200
@@ -96,7 +96,7 @@
 extern const char *archdep_home_path(void);
 
 /* Define the default system directory (where the ROMs are).  */
-#define LIBDIR          PREFIX "/lib/vice"
+#define LIBDIR          PREFIX "/share/vice"
 #define DOCDIR          LIBDIR "/doc"
 #define VICEUSERDIR     ".vice"
 
--- vice-1.21/configure.in~	2007-03-04 22:58:50.000000000 +0100
+++ vice-1.21/configure.in	2007-07-06 20:00:05.000000000 +0200
@@ -1891,7 +1891,7 @@
 		   [Where do we want to install the executable?])
 
 dnl Path for keymaps, ROMs etc.
-VICEDIR="$PREFIX/lib/vice";
+VICEDIR="/usr/share/vice";
 AC_SUBST(VICEDIR)
 
 if false ; then
--- vice-1.21/data/fonts/Makefile.am~	2006-01-30 22:48:17.000000000 +0100
+++ vice-1.21/data/fonts/Makefile.am	2007-07-06 21:34:07.000000000 +0200
@@ -1,4 +1,4 @@
-fontsdir = $(VICEDIR)/fonts
+fontsdir = /usr/share/fonts/misc
 
 fonts_DATA = \
 	vice-cbm.bdf
@@ -7,7 +7,6 @@
 
 install: install-data-am
 	@echo "Preparing fontdir..." ; \
-	cd $(DESTDIR)$(fontsdir) ; \
+	cd $(DESTDIR)$(datadir)/fonts/misc ; \
 	bdftopcf -o vice-cbm.pcf vice-cbm.bdf ; \
-	rm vice-cbm.bdf ; \
-	mkfontdir .
+	rm vice-cbm.bdf
