--- vice-1.21/src/arch/unix/archdep.c~	2007-02-25 18:31:39.000000000 +0100
+++ vice-1.21/src/arch/unix/archdep.c	2007-07-15 15:41:55.000000000 +0200
@@ -121,6 +121,14 @@
 #endif
 }
 
+const char *archdep_home_etc_path(void)
+{
+    char *home_etc = getenv("HOME_ETC");
+
+    if (home_etc) return home_etc;
+               return archdep_home_path();
+}
+
 char *archdep_default_sysfile_pathlist(const char *emu_id)
 {
     static char *default_path;
@@ -134,7 +142,7 @@
         const char *home_path;
 
         boot_path = archdep_boot_path();
-        home_path = archdep_home_path();
+        home_path = archdep_home_etc_path();
 
         /* First search in the `LIBDIR' then the $HOME/.vice/ dir (home_path)
            and then in the `boot_path'.  */
@@ -222,7 +230,7 @@
 {
     const char *home;
 
-    home = archdep_home_path();
+    home = archdep_home_etc_path();
     return util_concat(home, "/.vice/vicerc", NULL);
 }
 
@@ -230,7 +238,7 @@
 {
     const char *home;
 
-    home = archdep_home_path();
+    home = archdep_home_etc_path();
     return util_concat(home, "/.vice/fliplist-", machine_name, ".vfl", NULL);
 }
 
@@ -240,7 +248,7 @@
     const char *home;
     char *viceuserdir;
 
-    home = archdep_home_path();
+    home = archdep_home_etc_path();
 
     viceuserdir = util_concat(home, "/.vice", NULL);
 
--- vice-1.21/src/arch/unix/archdep.h~	2007-07-15 15:37:48.000000000 +0200
+++ vice-1.21/src/arch/unix/archdep.h	2007-07-15 15:42:59.000000000 +0200
@@ -93,7 +93,7 @@
 /* Ethernet default device */
 #define ARCHDEP_ETHERNET_DEFAULT_DEVICE "eth0"
 
-extern const char *archdep_home_path(void);
+extern const char *archdep_home_etc_path(void);
 
 /* Define the default system directory (where the ROMs are).  */
 #define LIBDIR          PREFIX "/share/vice"
--- vice-1.21/src/arch/unix/gui/uicommands.c~	2007-02-11 23:57:19.000000000 +0100
+++ vice-1.21/src/arch/unix/gui/uicommands.c	2007-07-15 15:44:24.000000000 +0200
@@ -295,7 +295,7 @@
 
 static UI_CALLBACK(load_quicksnap)
 {
-    char *fname = util_concat(archdep_home_path(), "/", VICEUSERDIR, "/",
+    char *fname = util_concat(archdep_home_etc_path(), "/", VICEUSERDIR, "/",
         machine_name, ".vsf", NULL);
 
     if (!ui_emulation_is_paused())
@@ -327,7 +327,7 @@
 
 static UI_CALLBACK(save_quicksnap)
 {
-    char *fname = util_concat(archdep_home_path(), "/", VICEUSERDIR, "/",
+    char *fname = util_concat(archdep_home_etc_path(), "/", VICEUSERDIR, "/",
         machine_name, ".vsf", NULL);
 
     interrupt_maincpu_trigger_trap(save_snapshot_trap, (void *)fname);
