--- vice-2.2/configure.in.orig	2011-02-21 16:41:42.186971716 +0100
+++ vice-2.2/configure.in	2011-02-21 18:43:03.414973392 +0100
@@ -1106,19 +1106,6 @@
 
   if test x"$UNIX_NETWORK_HEADERS_PRESENT" = "xyes"; then
 
-    dnl Check for possible extra needed network libraries
-    AC_CHECK_LIB(nsl, gethostbyname,[ NETPLAY_LIBS="-lnsl $NETPLAY_LIBS";
-                                      LIBS="-lnsl $LIBS"],,)
-    AC_CHECK_LIB(socket, gethostbyname,[ NETPLAY_LIBS="-lsocket $NETPLAY_LIBS";
-                                         LIBS="-lsocket $LIBS"],,)
-    AC_CHECK_LIB(bsd, gethostbyname,[ NETPLAY_LIBS="-lbsd $NETPLAY_LIBS";
-                                      LIBS="-lbsd $LIBS"],,)
-    AC_CHECK_LIB(net, gethostbyname,[ NETPLAY_LIBS="-lnet $NETPLAY_LIBS";
-                                      LIBS="-lnet $LIBS"],,)
-    AC_CHECK_LIB(inet, gethostbyname,[ NETPLAY_LIBS="-linet $NETPLAY_LIBS";
-                                       LIBS="-linet $LIBS"],,)
-    AC_CHECK_LIB(watt, gethostbyname,[ NETPLAY_LIBS="-lwatt $NETPLAY_LIBS";
-                                       LIBS="-lwatt $LIBS"],,)
 
     dnl Check for needed functions
     UNIX_NETWORK_FUNCS_PRESENT=yes
@@ -2040,15 +2027,6 @@
     LINK_X_LIBS="$GTK_LIBS $GTKGL_LIBS $LINK_X_LIBS"
   else
     AC_DEFINE(USE_COLOR_MANAGEMENT,,[Enable new color management code.])
-    dnl instead of assuming that libsocket needs libnsl, we
-    dnl will check the existance of libnsl first.
-
-    AC_CHECK_LIB(nsl, gethostbyname,[ X_LIBS="-lnsl $X_LIBS";
-                 LINK_X_LIBS="-lnsl $LINK_X_LIBS"],,$X_LIBS)
-    AC_CHECK_LIB(socket, gethostbyname,[ X_LIBS="-lsocket $X_LIBS";
-                 LINK_X_LIBS="-lsocket $LINK_X_LIBS"],,$X_LIBS)
-    AC_CHECK_LIB(bsd, gethostbyname,[ X_LIBS="-lbsd $X_LIBS";
-                 LINK_X_LIBS="-lbsd $LINK_X_LIBS"],,$X_LIBS)
     if test x"$ac_cv_lib_ICE_IceConnectionNumber" != "xyes"; then
       unset ac_cv_lib_ICE_IceConnectionNumber
     fi
