--- vice-2.2/data/fonts/Makefile.am.orig	2011-02-20 15:01:38.490973392 +0100
+++ vice-2.2/data/fonts/Makefile.am	2011-02-20 21:21:36.078979259 +0100
@@ -6,20 +6,3 @@
 
 EXTRA_DIST = $(fonts_DATA) cbm-directory-charset.fon 
 
-install: install-data-am
-@HAVE_FC_CACHE_TRUE@@SDL_COMPILE_FALSE@	@echo "Preparing fontdir, please wait..." ; \
-@HAVE_FC_CACHE_TRUE@@SDL_COMPILE_FALSE@	if [ ! `fc-cache --list > /dev/null 2>&1` ] ; then \
-@HAVE_FC_CACHE_TRUE@@SDL_COMPILE_FALSE@	  if test "x${HOME}" != "x" ; then \
-@HAVE_FC_CACHE_TRUE@@SDL_COMPILE_FALSE@	    mkdir ${HOME}/.fonts > /dev/null 2>&1 ; \
-@HAVE_FC_CACHE_TRUE@@SDL_COMPILE_FALSE@	    cp $(fontsdir)/CBM.ttf ${HOME}/.fonts ; \
-@HAVE_FC_CACHE_TRUE@@SDL_COMPILE_FALSE@	    fc-cache ; \
-@HAVE_FC_CACHE_TRUE@@SDL_COMPILE_FALSE@	  else \
-@HAVE_FC_CACHE_TRUE@@SDL_COMPILE_FALSE@	    echo '$$HOME not set... giving up.' ; \
-@HAVE_FC_CACHE_TRUE@@SDL_COMPILE_FALSE@	  fi \
-@HAVE_FC_CACHE_TRUE@@SDL_COMPILE_FALSE@	else \
-@HAVE_FC_CACHE_TRUE@@SDL_COMPILE_FALSE@	  echo 'fc-cache not available... giving up.' ; \
-@HAVE_FC_CACHE_TRUE@@SDL_COMPILE_FALSE@	fi
-@SDL_COMPILE_FALSE@	@echo "Preparing fontdir..." ; \
-@SDL_COMPILE_FALSE@	cd $(DESTDIR)$(fontsdir) ; \
-@SDL_COMPILE_FALSE@	bdftopcf -o vice-cbm.pcf vice-cbm.bdf ; \
-@SDL_COMPILE_FALSE@	${MKFONTDIR_BIN} .
